lib/pytest-lsp/pytest_lsp/client.py

```diff
@@ -82,6 +82,22 @@ def __init__(self, *args, configuration: Optional[Dict[str, Any]] = None, **kwar
         self._last_log_index = 0
         """Used to keep track of which log messages correspond with which test case."""
 
+        self._stderr_forwarder: Optional[asyncio.Task] = None
+        """A task that forwards the server's stderr to the test process."""
+
+    async def start_io(self, cmd: str, *args, **kwargs):
+        await super().start_io(cmd, *args, **kwargs)
+
+        # Forward the server's stderr to this process' stderr
+        if self._server and self._server:
```

Comment:
Checking `self._server` twice. Is this a typo?

```suggestion
        if self._server:
```

Reply:
Ah, good spot that should be `self._server and self._server.stderr:`